src/style.css

```diff
@@ -0,0 +1,139 @@
+/* #region page */
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+/* #endregion */
+
+/* #region imac */
+.card__imac-logo {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.code {
+  text-align: left;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+  margin-left: 16px;
+}
+/* #endregion */
+
+/* #region stars */
+.card__grade {
+  display: flex;
+  justify-content: space-between;
+  margin: 0  16px 24px;
+}
+
+.stars {
```

Comment:
you have to create a separate file for stars style according to the checklist:
![image](https://user-images.githubusercontent.com/22824431/221571839-7ddca35a-0b1b-45fb-9281-f78892fc6bc4.png)